docs/book/src/development/amis.md

```diff
@@ -0,0 +1,29 @@
+# Publish AMIs
+
+Publishing new AMIs is currently a manual process but it will be automated in th every near future (see [this issue](https://github.com/kubernetes-sigs/cluster-api-provider-aws/issues/1982) for progress).
+
+## Pre-reqs
+
+- You must have admin access to the CNCF AWAS account used for the AMIs (819546954734)
+
+## Process
+
+1. Clone [image-builder](https://github.com/kubernetes-sigs/image-builder)
+2. Open a terminal
+3. Set the AWS environment variables for the AMI account
+4. Change directory into `images/capi`
+5. Install dependencies by running:
+
+```shell
+make deps-ami
+```
+
+6. Build the AMIs using:
+
+```shell
+make build-ami-ubuntu-2004
```

Comment:
How long should we support 20.04?

Reply:
As discussed on slack we've dropped this, so just 22.04 and 24.04.